diagnostic.md

```diff
@@ -7,40 +7,43 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:prankmode/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+Light a candle to the gods?
```

Comment:
Wouldn't hurt, and you do indeed need to `cd` into the repo. However, the answer we were looking for was 'create a branch'.